### ISSUE TITLE
Octupole element creator not as Sextupole

### ISSUE THREAD

**Issue by simoneliuzzo** [atcollab/at#465]
it is not possible to create an octupole as 

`Octupole('OCT',length_oct,K_oct)`

while it is possible to create a Sextupole in this way

`Sextupole('OCT',length_sext,K_sext)`

**Comment by lfarv** [comment]
Exact. `Octupole` needs `PolynomA` and `PolynomB`. It should be possible to additionally allow the syntax you propose by checking the number of inout arguments.

**Comment by TeresiaOlsson** [comment]
Is there a reason for not changing the syntax of the octupole? We are using the octupole element in pyAT and it is strange for the users that this element doesn't follow the same syntax as other elements.
